Scripts/Python/grsnWallPython.py

```diff
@@ -44,1203 +44,773 @@
+pTubeClose = ptAttribNamedResponder(35, "Purple tube close", netForce=1)
+yTubeClose = ptAttribNamedResponder(36, "Yellow tube close", netForce=1)
 
-sTubeExclude = ptAttribExcludeRegion(45,"s tube exclude")
-nTubeExclude = ptAttribExcludeRegion(46,"n tube exclude")
+pTubeEntry = ptAttribNamedActivator(37, "Purple tube entry trigger")
+yTubeEntry = ptAttribNamedActivator(38, "Yellow tube entry trigger")
 
-sTeamWarpPt = ptAttribSceneobject(47,"s team warp point")
-nTeamWarpPt = ptAttribSceneobject(48,"n team warp point")
+pTubeMulti = ptAttribBehavior(43, "Purple tube entry multi", netForce=0)
+yTubeMulti = ptAttribBehavior(44, "Yellow tube entry multi", netForce=0)
 
-sTeamWin = ptAttribActivator(49,"s team win")
-nTeamWin = ptAttribActivator(50,"n team win")
+pTubeExclude = ptAttribExcludeRegion(45, "Purple tube exclude")
+yTubeExclude = ptAttribExcludeRegion(46, "Yellow tube exclude")
 
-sTeamQuit = ptAttribActivator(51,"s team quit")
-nTeamQuit = ptAttribActivator(52,"n team quit")
+pTeamWarpPt = ptAttribSceneobject(47, "Purple team warp point")
+yTeamWarpPt = ptAttribSceneobject(48, "Yellow team warp point")
 
-sTeamWinTeleport = ptAttribSceneobject(53,"s team win point")
-nTeamWinTeleport = ptAttribSceneobject(54,"n team win point")
+pTeamWin = ptAttribActivator(49, "purple team win")
+yTeamWin = ptAttribActivator(50, "yellow team win")
 
-nQuitBehavior = ptAttribBehavior(55,"s quit behavior")
-sQuitBehavior = ptAttribBehavior(56,"n quit behavior")
+pTeamQuit = ptAttribActivator(51, "purple team quit")
+yTeamQuit = ptAttribActivator(52, "yellow team quit")
 
-# sfx responders
+pTeamWinTeleport = ptAttribSceneobject(53, "purple team win point")
+yTeamWinTeleport = ptAttribSceneobject(54, "yellow team win point")
 
-nPanelSound = ptAttribResponder(57,"n panel sound",['main','up','down','select','blockerOn','blockerOff','gameStart','denied'],netForce=1)
-sPanelSound = ptAttribResponder(58,"s panel sound",['main','up','down','select','blockerOn','blockerOff','gameStart','denied'],netForce=1)
+yQuitBehavior = ptAttribBehavior(55, "yellow quit behavior", netForce=0)
+pQuitBehavior = ptAttribBehavior(56, "purple quit behavior", netForce=0)
 
+#sfx responders
+yPanelSound = ptAttribResponder(57,"yellow panel sound",['main','up','down','select','blockerOn','blockerOff','gameStart','denied'],netForce=0)
+pPanelSound = ptAttribResponder(58,"purple panel sound",['main','up','down','select','blockerOn','blockerOff','gameStart','denied'],netForce=0)
 
+#TOC specific
+panelLightAnimP = ptAttribMaterialAnimation(61, "Purple Panel Light Animation")
+panelLightAnimY = ptAttribMaterialAnimation(62, "Yellow Panel Light Animation")
+sitPanelLightsY = ptAttribSceneobjectList(63, "Yellow sitting indicators")
+sitPanelLightsP = ptAttribSceneobjectList(64, "Purple sitting indicators")
+confirmPanelLightsY = ptAttribSceneobjectList(65, "Yellow ready indicators")
+confirmPanelLightsP = ptAttribSceneobjectList(66, "Purple ready indicators")
```

Comment:
validate these and change everything back to north/south to maintain consistency with named python files (or rename those in the PFMs)